src/MemberProfilePage.js

```diff
@@ -0,0 +1,23 @@
+import gql from "graphql-tag";
+import { graphql } from "react-apollo";
+import MemberProfile from './MemberProfile';
+
+const userQuery = gql`
+  query getUser($id: ID) {
+    User(id: $id) {
+      picture
+      name
+      description
+      githubName
+      flowdockName
+      email
```

Comment:
kind of a tangent, but I wonder if people might be hesitant to have their emails listed + indexable on the web